app/components/UI/FiatOnRampAggregator/Views/AmountToBuy.tsx

```diff
@@ -659,11 +674,23 @@ const AmountToBuy = () => {
               'fiat_on_ramp_aggregator.no_tokens_available',
               {
                 network: NETWORKS_NAMES[selectedChainId],
+                region: selectedRegion?.name,
               },
             )}
-            ctaOnPress={() => navigation.goBack()}
+            ctaLabel={strings('fiat_on_ramp_aggregator.try_different_region')}
+            ctaOnPress={() => {
+              toggleRegionModal();
+            }}
```

Comment:
```suggestion
            ctaOnPress={toggleRegionModal}
```

Reply:
If I do this change typescript will complain "Type 'Function' is not assignable to type '() => void'" I have changed it to 
```suggestion
            ctaOnPress={() => toggleRegionModal()}
```